src/Avalonia.Controls/WindowTransparencyLevel.cs

```diff
@@ -20,6 +20,11 @@ public enum WindowTransparencyLevel
         /// <summary>
         /// The window background is a blur-behind with a high blur radius. This level may fallback to Blur.
         /// </summary>
-        AcrylicBlur
+        AcrylicBlur,
+        
+        /// <summary>
+        /// The window background is based on desktop wallpaper tint with a blur. This will only work on Windows 11 
+        /// </summary>
+        Mica
```

Comment:
I am not sure about having very platform specific value in here.
Even acrylic exists at least in macOS.

Reply:
AFAIK AcrylicBlur doesn't work rn in Win11. Was it completely replaced by Mica?

Reply:
@maxkatz6 I know it's a little weird, but right now `WindowTransparencyLevel` is not really cross-platform, the actual value used by each `WindowImpl` is different depending on the situation.
Also, I think that this effect can be implemented in the future as a cross platform effect once new renderer will be available.
As for AcrylicBlur on Win11, it works if you don't use `BorderOnly` as WindowDecoration (which call some DWM API that was changed to creates a static accent color on window background).

Reply:
TBH should have separated transparency from backdrop effects.
1) gracefull fallback to regular transparency if blur effects aren't available isn't possible with the current API
2) if we keep adding more stuff to that enum it becomes more and more weird

I think I'm ok with adding `Mica` as enum member for 0.10.x timeframe, but for the next version we need separate  `PreferredBackdropDecorationEffects` (`None`, `AcryllicBlur`, `Mica`, `Blur`) and `WantsTransparency` (`true`, `false`, takes effect when effective backdrop decoration is `None`)